src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
```

Comment:
The variable name 'result' is very generic. Consider using a more descriptive name that reflects the content of the variable, such as 'styleObject' or 'cssProperties'.